### DISCUSSION TITLE
Is it possible to add a `0:00` string in meeting notes/description, to enable youtube to create video chapters?

### DISCUSSION THREAD

**Discussion by TomSweeneyRedHat** [containers/podman#21771] [Ideas]
Created by @isolcman in a reply to #11635, I thought it would stand up better in it's own discussion.

Hi, Is it possible to add a `0:00` string in meeting notes/description, to enable youtube to create video chapters?

See random podman video description - the only thing needed is additional item straight below the "Topics:", containing string "0:00" (it might be something like `* Intro (0:00 in the video)`

![obraz](https://github.com/containers/podman/assets/4583553/2a57199a-4518-415b-9b0f-1a7c4d13452b)

_Originally posted by @lstolcman in https://github.com/containers/podman/discussions/11635#discussioncomment-8522599_

**Comment by TomSweeneyRedHat** [comment]
TomSweeneyRedHat
Feb 19, 2024
Maintainer Author

I was unaware of that. I'll go give it a go in the last community meeting recording and will see how it goes. Thanks a bunch for the tip @lstolcman !
@lstolcman
lstolcman
Feb 20, 2024

Seems that, it works:

KRIpvdD4v5Gw

the description in a video is a bit crippled though. As I can see in the other youtube videos, the default format is like

`<timestamp> - <description>`

and then youtube catches it. Is it possible to have such format?

![image](https://github.com/containers/podman/assets/25613829/30cd8284-71f7-4386-936c-fb3857355bdc)


obraz

**Comment by TomSweeneyRedHat** [comment]
@lstolcman please checkout https://www.youtube.com/watch?v=1wOoZ5qPeII and let me know if that's what you were thinking about.

**Reply by lstolcman** [reply]
Yes, that's exactly it, and it helps a lot. Thank you!

**Reply by TomSweeneyRedHat** [reply]
Super, I'll see if I can convert the older ones over time and will try to remember to use this format going forward.  Thanks again for the tip!
